packages/server/postgres/helpers/OrgUserAudit.ts

```diff
@@ -0,0 +1,16 @@
+import {insertOrgUserAudit as insertQuery} from '../queries/OrgUserAudit'
```

Comment:
can we keep the file name == function name & export it as the default? doing this means we don't have to rename generated query when we import it

Reply:
@mattkrick hmm yeah that would be nice. if i'm not mistaken, i don't think there's any way to configure pgtyped to export a generated fn as the default. i think the intention of the author was for each sql file to contain multiple queries pertaining to one table.